src/components/person_info.tsx

```diff
@@ -8,16 +8,16 @@ export default function PersonInfo(props: {
     description: string;
     extraClasses?: string;
 }) {
-    const baseClassNames = `w-[90vw] sm:w-[80vw] md:w-[70vw] shadow-sm whitespace-pre-line shadow-wai-gray bg-pure-white text-xl sm:text-xl md:text-2xl font-mono font-bold text-center text-wai-gray border-4 rounded-lg border-wai-gray p-4 justify-self-center flex items-center`;
-    const cardClassNames = `${baseClassNames} ${props.extraClasses || ""}`;
-    const tagClassNames = `${props.tagColour ? `!${props.tagColour}` : "text-wai-gray"}`;
+    const baseClassNames = `w-full shadow-sm whitespace-pre-line shadow-wai-gray bg-pure-white text-xl sm:text-xl md:text-2xl font-mono font-bold text-center text-wai-gray border-4 rounded-lg border-wai-gray p-2 md:p-4 justify-self-center`;
```

Comment:
i might look into getting these styles cleaned up at some point, but that would be a lot of config alterations